code/go/0chain.net/blobbercore/blobbergrpc/proto/blobber_service.proto

```diff
@@ -0,0 +1,118 @@
+syntax = "proto3";
+
+package blobber;
+
+option go_package = "code/go/0chain.net/blobbercore/blobbergrpc";
+
+
+import "blobber_contract.proto";
+import "google/api/annotations.proto";
+
+
+service BlobberService {
+  rpc GetAllocation(GetAllocationRequest) returns (GetAllocationResponse) {
+    option (google.api.http) = {
+      get: "/v2/allocation"
+    };
+  }
+  rpc GetFileMetaData(GetFileMetaDataRequest) returns (GetFileMetaDataResponse) {
```

Comment:
Could you add comment for each method with input/output?

Reply:
The comment rule works for exported golang functions, not for the protos. The generated stubs already have a comment on them.